deploy/skaffold/Dockerfile

```diff
@@ -14,35 +14,35 @@
 
 # Download kubectl
 FROM alpine:3.10 as download-kubectl
-ENV KUBECTL_VERSION v1.12.8
+ENV KUBECTL_VERSION v1.17.1
```

Comment:
I don't think we want to jump directly to the latest version of kubectl. Only a few users will have this version installed and it's not guaranteed to be compatible with every older version of k8s. ping @nkubala @balopat

Reply:
That makes sense.  How about 1.14?

It's also possible to install kubectl as a google cloud sdk component, which will provide a consistent version without needing it to be specified separately from the cloud sdk version.  Currently the sdk provides version 1.14.x of `kubectl`.  It also includes binaries like `kubectl.1.1{3,4,5}`, but I don't see a way for skaffold to use those.